discovery_child_development/utils/io.py

```diff
@@ -0,0 +1,24 @@
+"""
+Various input output utilities
+"""
+from discovery_child_development import PROJECT_DIR, logging
+from yaml import safe_load
+from typing import TypeVar, Iterator
+import pathlib
+import pickle
+import json
+import os
+import pandas as pd
+from zipfile import ZipFile
+
+PathLike = TypeVar("PathLike", str, pathlib.Path, None)
+
+DEF_CONFIG_PATH = PROJECT_DIR / "discovery_child_development/config"
+VERBOSE = True
```

Comment:
I guess we're not using this for now - do you want to keep it for future use cases?

Reply:
I just copied this script from the Innovation Sweet Spots repo because I wanted to use the config function! Which bit do you mean? The `VERBOSE = True` line?